src/main/java/com/rarchives/ripme/ripper/rippers/EroShareRipper.java

```diff
@@ -79,12 +91,14 @@ public Document getNextPage(Document doc) throws IOException {
         // Find next page
         String nextUrl = "";
         Element elem = doc.select("li.next > a").first();
-        logger.info(elem);
```

Comment:
I assume I did it because I was trying to debug and it was clogging up the log/output